kpops/utils/docstring.py

```diff
@@ -74,6 +63,14 @@ def _trim_description_end(desc: str) -> str:
         log.debug("Returned an empty string.", exc_info=True)
         return ""
 
+    desc_enders: list[str] = [
```

Comment:
just fyi: type annotation is not necessary here as it can be easily inferred by the type checker